Client/game_sa/gamesa_init.h

```diff
@@ -104,3 +104,58 @@ void MemOrFast(U ptr, const T value)
 
 bool GetDebugIdEnabled(uint uiDebugId);
 void LogEvent(uint uiDebugId, const char* szType, const char* szContext, const char* szBody, uint uiAddReportLogId = 0);
+
+struct __THISCALL{};
+struct __CDECL{};
+struct __STDCALL{};
+struct __FASTCALL{};
+struct __VECTORCALL{};
```

Comment:
It would be nice to organize the new functionality into a new header.

Reply:
What do you propose?

Reply:
Put all changes into a new file

Reply:
Yeah, but I mean the name. What could I name the new file? Personally, I have no idea